spec/system/support_interface/see_candidates_spec.rb

```diff
@@ -3,16 +3,25 @@
 RSpec.feature 'See candidates' do
   include DfESignInHelpers
 
-  scenario 'Support agent visits the list of candidates' do
+  scenario 'Support agent visits the list of candidates and selects one who has never signed in' do
     given_i_am_a_support_user
     and_there_are_candidates_in_the_system
     and_i_visit_the_support_candidate_page
     then_i_should_see_the_candidates
 
-    when_i_click_on_a_candidate
+    when_i_click_on_a_candidate_with_no_applications
     then_i_see_the_candidate_details
   end
 
+  scenario 'Support agent selects a candidate with one application' do
```

Comment:
The goal is to have a single scenario per file (we should still document why - speed & readability), can this be merged with the other scenario?

Reply:
Fair enough – fixed and force pushed.